localenv/mock-account-servicing-entity/app/lib/apiClient.ts

```diff
@@ -8,6 +8,8 @@ export const StepNames = {
   endInteraction: 3
 }
 
+const IDP_SECRET = '2pEcn2kkCclbOHQiGNEwhJ0rucATZhrA807HTm2rNXE='
```

Comment:
can we pass this through `process.env`?

Reply:
it was a bit involved but yes [15178f6](https://github.com/interledger/rafiki/pull/2746/commits/15178f6f23941359e450d97199c3aa4d5845696a)